examples/nrf24l01_btle/nrf24l01_btle.c

```diff
@@ -219,16 +219,16 @@ int nrf24_cfg(int fd)
   int32_t txpow = DEFAULT_TXPOWER;
 
   /**************************************************************************
-   * if using RATE_1Mbps from include/nuttx/wireless/nrf24l01.h,
+   * if using RATE_1MBPS from include/nuttx/wireless/nrf24l01.h,
```

Comment:
please remove the comment as it is not relevant anymore

Reply:
Done.